chutney/server/src/main/java/com/chutneytesting/security/api/UserController.java

```diff
@@ -41,4 +60,21 @@ public UserDto currentUser(HttpServletRequest request, HttpServletResponse respo
     public UserDto loginForwardUser(HttpServletRequest request, HttpServletResponse response) {
         return this.currentUser(request, response);
     }
+
+    @PostMapping("/login")
+    public ResponseEntity<?> login(@RequestBody LoginRequest loginRequest) {
```

Comment:
You deleted the formlogin from security config but use nevertheless a login endpoint who 's the same.